### ISSUE TITLE
Option for Motion Constrained Tile Set (MCTS) that also enable motion vector constraint for inter prediction?

### ISSUE THREAD

**Issue by yinzhifan** [ultravideo/kvazaar#268]
Hi,
I am doing a research about VR FoVe profile using MCTS, and currently I am trying to split the tiles on video encoding using MCTS and playback each tile separately (so a bitstream in combination of arbitrary tiles can be decoded). When (after dividing each tile to one track) using GPAC client to playback a single tile track (it's miracle that should work), I noticed there are glitches in the tile boundaries, which after some research I realized that is because KVAZAAR still use un-constrained inter-predictions for tiles, thus missing tiles will sometimes cause such glitches. No related issue is found in this tab, and I wonder is there an option to enable motion constraint on inter prediction?
The commands I used : 
kvazaar --input @filename.yuv --input-res @input_resolution --input-fps @fps  + below + -o @output_name
--tiles 4x2 --slices tiles --mv-constraint frametilemargin

Thanks in advance..

![pic1](https://user-images.githubusercontent.com/18151050/83398636-4ddef980-a432-11ea-8ad2-7a92a34676d1.png)

![pic2](https://user-images.githubusercontent.com/18151050/83398639-4fa8bd00-a432-11ea-9b18-0c181b4c63a8.png)

**Comment by Arizer** [comment]
Hello and thanks for the report.

The options you used should work.

A motion vector constraint check was missing from one of the more recently changed part of the inter prediction code. The missing check was added and pushed to master (d1abf85). Hopefully this solves your issue. At least I did not immediately notice problems when playing with GPAC client.

We should probably set some kind of automated test for this use case.
